src/components/qcResultsUpload/QcResultsUploadForm.vue

```diff
@@ -71,18 +72,27 @@ export default {
       usedBySelected: 'extraction',
       busy: null,
       disableUpload: null,
+      uploadSuccessful: null,
```

Comment:
just a minor thing, May be It would be better to have a comment here to avoid ambiguity in future that  describes the expected  states - true, false and null

Reply:
Thank you, I've added a comment